### ISSUE TITLE
PCB file might be corrupted?

### ISSUE THREAD

**Issue by sortofsleepy** [ai03-2725/Unified-Daughterboard#4]
👋 

First off thanks to all for making this open source!

**Tldr'** - When opening the PCB file in KiCad, I get the following error. 
![daughterboard](https://user-images.githubusercontent.com/308302/80055298-3416d200-84d6-11ea-8039-902573f89480.JPG)

The weird thing is I have the latest version, 5.1.5 at the time of this writing.

======

Decided to start learning about PCB design; wanted to prototype something using a daughter board, came across this, looks perfect as is. Since I have 0 skill in SMD soldering nor the tooling I was hoping to get the fab house to assemble everything with parts. JLCPCB seems like the place to do it since they'll read everything off the gerber file and offer the service. 

The issue is that for the assembly service, the project needs to be at a minimum 20x20 so I just wanted to tweak the edge cuts just a bit. 

I cloned with `--recursive` so I believe I got everything, is there possibly something I'm still missing?

Also open to alternate places that could do the work, just looking at JLCPCB cause I've heard of em and they the easier interface to deal with.

Thanks so much 🎉

**Comment by ai03-2725** [comment]
The file was modified with a fairly recent nightly copy of KiCad; older versions may have gotten "locked out" due to KiCad preventing older versions from failing to load newer formats.
